src/redux/missions/missions.js

```diff
@@ -1 +1,27 @@
-/* some comment */
+import fetchMissions from '../../helpers/fetchMissions';
+
+const GET_MISSIONS = 'missionStore/missions/GET_MISSIONS';
+
```

Comment:
[Optional]
I think we should follow the Ducks convention when we define types. It should look like ` const CREATE = 'my-app/widgets/CREATE';`
[Example Link](https://github.com/erikras/ducks-modular-redux#the-proposal)